packages/gestalt/src/ExpandableModule.js

```diff
@@ -0,0 +1,45 @@
+// @flow strict
+import React, { useState, type Node } from 'react';
+import Box from './Box.js';
+import Divider from './Divider.js';
+import ExpandableModuleBase from './ExpandableModuleBase.js';
+import styles from './ExpandableModule.css';
+
+type Props = {|
+  items: Array<{|
+    title: string,
+    icon?: string,
+    iconAccessibilityLabel?: string,
+    summary?: Array<string>,
```

Comment:
Does summary need to be an array of strings? Can summary be a node?

Reply:
Based on the discussion in TDD meeting, we decided to use array of strings instead of Node. We may revisit it on next meeting